source/2019-10-31-octane-release-update.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Octane Release Update
+author: Yehuda Katz, Matthew Beale
+tags: Recent Posts, 2019, Announcement
+responsive: true
+---
+
+## Octane Release Update
```

Comment:
```suggestion

```

The title is added automatically. This line should be removed or we have duplicates